### PR TITLE
hard-wrap all pre sections to 80 columns

### PR DESCRIPTION
**Pull request by pmuellr**
To make the various examples fit without scrolling on a small display (eg, tablet), I hard-wrapped all the `<pre>` sections so they were no more than 80 characters wide.

I have a PDF suitable for reading on a tablet, here:

https://www.dropbox.com/s/274qvd42px40rzf/Mostly%20adequate%20guide%20to%20FP%20%28in%20javascript%29.pdf?dl=0

### CODE REVIEW

File: ch5.md [hunk 1]
```diff
@@ -227,11 +231,10 @@ var f = function(x){ return x === 4; };
 var isFourLetterWord = compose(f, g);
 ```
 
-**A distinguished morphism called identity**
-Let's introduce a useful function called `id`. This function simply takes some input and spits it back at you. Take a look:
+**A distinguished morphism called identithpintroduce a useful function called `id`. This function simply takes some input and spits ihpke a look:
```
Comment (DrBoolean) [inline ch5.md]: Just noticed something odd here is all. Maybe some spaces got deleted or something?

**Comment by DrBoolean** [comment]
Wow, thanks so much for the effort! I just noticed a small blip in ch5, but everything else looks great.

File: ch5.md (hunk 1 above)
Comment (pmuellr) [inline ch5.md]: oh dear, some kind of finger check; shall I fix and re-issue the PR, or whatever, or do you want to handle it some other way.

**Comment by DrBoolean** [comment]
I think you can just push to the branch and the PR will update itself. Ugh, looks like there's a small merge conflict according to the warning

**Comment by pmuellr** [comment]
Lots of working going on in this repo!  Think I got everything merged.

**Comment by DrBoolean** [comment]
Wow, thanks!

**Comment by DrBoolean** [comment]
Hi! I ended up finding an issue in ch5 so I reverted just the 1 file. Something around `ihp` got funky. Rest looks great though so I didn't revert. For ch5, you might want to just open a new PR. Thanks again for the help!

**Pull request merged**